src/components/homepage/BannerCard.jsx

```diff
@@ -0,0 +1,21 @@
+import "../../styles/components/banner-card.css";
+
+const BannerCard=()=>{
+    return (
+        <>
+        <main className="main-section">
+        <div className="hero-container">
+            <h1 className="title">Fashion-hub Sale</h1>
+            <p>UPTO 50% OFF on various products</p>
+            <a href="/pages/products/products.html" className="shop-btn"><button className="btn-default btn-primary">Shop

```

Comment:
Remove a tag use Link or Navigate insted of a

Reply:
Thanks!! will add in next PR.